prisma/migrations/20220707204448_add_transaction_model/migration.sql

```diff
@@ -0,0 +1,14 @@
+-- CreateTable
+CREATE TABLE `Transaction` (
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `hash` VARCHAR(255) NOT NULL,
+    `amount` VARCHAR(255) NOT NULL,
```

Comment:
Shouldn't this be a decimal(16,2) ?

Reply:
Why `(16,2)`?  I think we could do `(16.0)` and then register the default base unit steps in the `Chain` table (this would be 2 for eCash and 8 for Bitcoin Cash, for example). The BCH lib we are using to get the data just return the amount in satoshis as a string, so it would also be easy to do the conversion and we only have to worry about the decimal places when displaying this info to the user.

Reply:
Actually, `Decimal(x, 0)` looks like an overkill, we could just user INTEGER in this case to save the satoshis, what do you think?

Reply:
Addressed.